shiny/http_server.py

```diff
@@ -0,0 +1,13 @@
+from flask import Flask
+
+
+app = Flask(__name__)
+
+
+@app.route('/v1/do-nothing/<unicorn_name>', methods=['POST'])
+def do_nothing(unicorn_name):
+    return ''
```

Comment:
This is cheating. Can we at least grab the path from the `.proto` file somehow? At the very least it is in `shiny_pb2.DESCRIPTOR.serialized_pb`, but in an opaque way.

Also, prefer this:

```python
from google.protobuf import empty_pb2
from google.protobuf import json_format

def do_nothing(unicorn_name):
    result = empty_pb2.Empty()
    return json_format.MessageToDict(result)
```